03-Raster-Calculations-In-R.Rmd

```diff
@@ -121,10 +125,18 @@ library(raster)
 
 #view info about the dtm & dsm raster data
 GDALinfo("NEON_RemoteSensing/HARV/DTM/HARV_dtmCrop.tif")
-GDALinfo("NEON_RemoteSensing/HARV/DTM/HARV_dsmCrop.tif")
+GDALinfo("NEON_RemoteSensing/HARV/DSM/HARV_dsmCrop.tif")
+
+```
+
+Both rasters are in the same CRS, have the same resolution and mins/maxes are
```

Comment:
and have defined minimum and maximum values. 

This would be a nice list rather than a sentence to break things up visually.